apps/admin-x-activitypub/src/views/Profile/Profile.tsx

```
@@ -2,230 +1,37 @@
-import {handleViewContent} from '@utils/content-handlers';
-
-import APAvatar from '@components/global/APAvatar';
-import ActivityItem from '@components/activities/ActivityItem';
-import FeedItem from '@components/feed/FeedItem';
-import FollowButton from '@components/global/FollowButton';
-import Layout from '@components/layout';
-import Separator from '@components/global/Separator';
-import ViewProfileModal from '@components/modals/ViewProfileModal';
-import {useFeatureFlags} from '@src/lib/feature-flags';
-import {useNavigate} from '@tryghost/admin-x-framework';
+interface ProfileProps {}
```

Biome (1.9.4)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)